### ISSUE TITLE
Constraint Strength not showing proper keyframe status

### ISSUE THREAD

**Issue by MangoQuaffer** [2d-inc/support#186]
Constraint Strength currently does not show proper keyframe status.  

If you key a the strength value on a constraint from 0 to 1 let's say, the **blue keyframe icon will show no matter where you drag on the timeline.** (Similarly if you do opposite way and take away a key and if you have multiple keys set to 1, you will not see correct status when navigating timeline)

You can deselect and reselect the constraint to see the proper keyframe status on that specific frame, but when you navigate the timeline, the keyframe status will not update and will remain as whatever the status was during the frame you were on when selecting the constraint.

When going to a spot on the timeline that is not keyed for the strength, it will display the proper status when you press the blue key icon.  **So you can properly set new keys and see the status correctly.**

However, since the status does not change, if you scrub while the icon is blue/set, and set a key, it will look like nothing happened unless you look at the timeline because the icon was on to begin with.

**In summary, constraint keys do not properly display while navigating timeline, but do display properly when setting new keys.**
